port/aix/omrosdump_helpers.c

```diff
@@ -132,6 +132,9 @@ genSystemCoreUsingGencore(struct OMRPortLibrary *portLibrary, char *filename)
 	BOOLEAN filenameWasEmpty = FALSE;
 	struct coredumpinfo coreDumpInfo;
 	int rc = -1;
+#if defined(DUMP_DBG)
+	int saved_errno;
+#endif
```

Comment:
comments for `#endif`: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif

```suggestion
#endif /* defined(DUMP_DBG) */
```